.github/workflows/linux.yml

```diff
@@ -0,0 +1,79 @@
+name: Linux
+
+on: [pull_request]
```

Comment:
Any reason for removing "push" here? I mean, according to our contributor guide we should send multiple versions, but sometimes do push on minor changes.

Reply:
Yes, it's intentional. if you update the PR with a force-push it will rebuild. 
The idea here is to avoid automatic builds every time you push something in your own fork or when someone forks stuff. As far I know all changes in avocado are merged by a PR, aren't they?